sdk/python/feast/dqm/profilers/ge_profiler.py

```diff
@@ -140,9 +141,12 @@ def analyze_dataset(self, df: pd.DataFrame) -> Profile:
         return GEProfile(expectation_suite=self.user_defined_profiler(dataset))
 
     def to_proto(self):
+        # keep only the code and drop context for now
+        # ToDo (pyalex): include some context, but not all (dill tries to pull too much)
```

Comment:
Curious what this means?

Reply:
So right now for easier deserialization I drop the whole context of user defined function and keep only its code. The opposite of that (what dill does by default) is include all globals and refer the original module, which won't be available at the runtime. But maybe in future somebody would want to use at least function clojure, so we might want to support that.